server/modules/elastalert/elastalert.go

```diff
@@ -355,12 +382,20 @@ func (e *ElastAlertEngine) startCommunityRuleImport() {
 			}
 		}
 
-		detections, errMap := e.parseRules(zips)
+		detections, errMap := e.parseZipRules(zips)
+		if errMap != nil {
+			log.WithField("error", errMap).Error("something went wrong while parsing sigma rule files")
+			continue
+		}
+
+		repoDets, errMap := e.parseRepoRules(allRepos)
 		if errMap != nil {
 			log.WithField("error", errMap).Error("something went wrong while parsing sigma rule files")
```

Comment:
Avoid using identical error strings, to better facilitate troubleshooting problems reported by customers.

Reply:
Updated to be more specific.